cypress/pages/homePage.js

```diff
@@ -23,14 +23,18 @@ export class homePage{
     cy.get(this.homePageSelectors.designServices).click();
    }
     
-   clickSingleVanities(){
-    cy.get(this.homePageSelectors.singleVanities).click()
+   navigateToFloatingBathroomVanities(){
```

Comment:
Can you rename this function liek clickFloatingBathroomVainities? Name convention is important :)

Reply:
done